bigtable-dataflow-parent/bigtable-hbase-beam/pom.xml

```diff
@@ -27,12 +27,34 @@ limitations under the License.
     and writers in Google Cloud Dataflow.
   </description>
 
+  <dependencyManagement>
```

Comment:
@igorbernstein2 PTAL at the changes made here - they now match what is in the main branch